gradle/scripts/README.md

```diff
@@ -177,23 +186,30 @@ apply from: "${rootDir}/gradle/scripts/build-flags.gradle"
 configure(projectsWithFlags('java')) {
     // Common dependencies
     dependencies {
-        compileOnly 'com.google.code.findbugs:jsr305'
-        compile 'com.google.guava:guava'
+      compileOnly libs.findbugs
+      implementation libs.guava
     }
 }
 
 // In case you need to get the version number of an artifact:
 println "Guava version: ${managedVersions['com.google.guava:guava']}"
+// Note that it is not recommended to use `managedVersions` with the module defined multiple times with 
+// different aliases. Because if a module is declared with different versions, the version returned by 
+// `managedVersions` is determined by how the version catalogs are indexed.
```

Comment:
I see, so it is possible that the version returned by `managedVersions` isn't necessarily equal to the eventually resolved version. Am I understanding correctly?

Reply:
`managedVersions` can be safely used  when a `groupId:artifactId` is declared only once in a whole `dependencies.toml`.

Let's say the following dependencies are defined.
```toml
[libraries.jsoup]
module = "org.jsoup:jsoup"
version = "1.15.1"

[libraries.thrift09]
module = "org.apache.thrift:libthrift"
version = "0.9.3-1"
[libraries.thrift012]
module = "org.apache.thrift:libthrift"
version = "0.12.0"
```

We can assume that `managedVersions['org.jsoup:jsoup']` returns `"1.15.1"`. However, we can't assert whether `managedVersions['org.jsoup:jsoup']` returns `"0.9.3-1"` or `"0.12.0"`.